### ISSUE TITLE
[BUG] 

### ISSUE THREAD

**Issue by Dabaduck** [realmazharhussain/gdm-settings#46]
### What happened?

I compiled the program following the instructions in the readme and when I try to apply any changes I get:
```
saving default theme ...
Traceback (most recent call last):
  File "/usr/local/lib/python3/dist-packages/gdm_settings/gdm_settings_gui.py", line 95, in on_apply
    if self.settings.apply_settings():
  File "/usr/local/lib/python3/dist-packages/gdm_settings/settings_manager.py", line 670, in apply_settings
    self.apply_gresource_settings()
  File "/usr/local/lib/python3/dist-packages/gdm_settings/settings_manager.py", line 578, in apply_gresource_settings
    self.command_elevator.add(f'update-alternatives --quiet --install {UbuntuGdmGresourceFile} {path.basename(UbuntuGdmGresourceFile)} {gresource_utils.CustomGresourceFile} 0')
NameError: name 'UbuntuGdmGresourceFile' is not defined

```
I have installed all the dependencies as indicated in the readme:
```
sudo apt install libadwaita-1-dev libglib2.0-dev python-gi-dev gettext #Runtime Dependencies
sudo apt install meson gobject-introspection #Build Dependencies
```

I tried to install the flatpak version as well, but I get the same error.
Now I would like to try uninstalling gdm-settings ("compiled version") and reinstalling it, but unfortunately I don't know how to do it.
Sorry I'm very new in the linux world.

### Expected behavior

It should change the login configuration, but it doesn't.

### To Reproduce

Open gdm-settings change a settings and click on the Apply button.

https://user-images.githubusercontent.com/68338584/167300108-57fc6ae2-c445-4591-93cf-e1625d262b87.mp4

### Application Version

Login Manager Settings v0.5.2

### Source

Source Code

### Environment

OS: Ubuntu 22.04
GNOME: 42

**Comment by realmazharhussain** [comment]
I have fixed the issue and created a new release v0.5.3 for the application. Update the application and it should work fine now.

**Comment by Dabaduck** [comment]
Now it seems to work. Thanks for adding uninstall instructions.
